Source/SelfService/Web/microservice/dataTable.tsx

```diff
@@ -28,25 +28,34 @@ const styles = {
     }
 };
 
-export const sortByRuntimeVersion = (params: GridValueGetterParams) => {
-    const runtimeVersion = params.row.edit.extra.runtimeImage.replace(/dolittle\/runtime:/gi, '');
+const sortByRuntimeVersion = (params: GridValueGetterParams) => {
+    const runtimeVersion = params.row.edit?.extra?.runtimeImage?.replace(/dolittle\/runtime:/gi, '');
+    if (runtimeVersion === undefined) return 'N/A';
+
     return `${runtimeVersion || 'None'}`;
 };
 
-const tooltipCell = (params: GridRenderCellParams) => (
-    // TODO: Tooltip needs public urls. Map them into title.
-    <Tooltip title={``} arrow>
-        <span>{params.row.edit.extra.isPublic ? 'Available' : 'None'}</span>
-    </Tooltip>
-);
+const tooltipCell = (params: GridRenderCellParams) => {
+    const publicUrl = params.row.edit?.extra?.isPublic;
+    if (publicUrl === undefined) return 'N/A';
+
+    return (
+        // TODO: Tooltip needs public urls. Map them into title.
+        <Tooltip title={``} arrow>
+            <span>{params.row.edit?.extra?.isPublic ? 'Available' : 'None'}</span>
+        </Tooltip>
+    );
```

Comment:
If we return just a string here, the styling might be different. I believe it is better to just change the text inside the tooltip and span:
```suggestion
    const hasPublicUrl = params.row.edit?.extra?.isPublic;

    return (
        // TODO: Tooltip needs public urls. Map them into title.
        <Tooltip title={``} arrow>
            <span>{
                hasPublicUrl === true ? 'Available' :
                hasPublicUrl === false ? 'None' :
                'N/A'
            }</span>
        </Tooltip>
    );
```